src/Microsoft.VisualStudio.ProjectSystem.Managed/Text/LazyStringSplit.cs

```diff
@@ -67,7 +67,7 @@ internal Enumerator(in LazyStringSplit split)
                 _index = 0;
                 _input = split._input;
                 _delimiter = split._delimiter;
-                Current = null;
+                Current = null!;
```

Comment:
I decided to allow this, as it's set only when the enumerator is in an invalid state...